public_dropin_gpu_environments/triton_server/start_server.sh

```diff
@@ -21,7 +21,7 @@ fi
 echo
 echo "Executing command: tritonserver --model-repository=${CODE_DIR}"
 echo
-nohup tritonserver --model-repository=${MODEL_DIR} &
+nohup tritonserver --model-repository=${CODE_DIR}/model_repository &
```

Comment:
the model-repository path is error-prone:
- Nemo names it as `model-store`
- Triton names it as `model_repository`
- Custom model tends to drop high level directory without any notice.

This should be fixed in a separate PR: users directory structure should be reconstructed, so Triton can read it. If it's not possible - raise an error.